### PR TITLE
Metrics and Resource-sidebar updated to pf4 Button component

### PR DESCRIPTION
**Pull request by sg00dwin**
Screen Shot 2019-10-08 at 4 19 23 PM

Screen Shot 2019-10-14 at 12 19 51 PM
Screen Shot 2019-10-14 at 12 20 13 PM

### CODE REVIEW

File: frontend/public/components/sidebars/resource-sidebar.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import { CloseIcon, DownloadIcon, InfoCircleIcon, PasteIcon } from '@patternfly/react-icons';
+import { Button } from '@patternfly/react-core';
```
Comment (rhamilto) [inline frontend/public/components/sidebars/resource-sidebar.jsx]: This file is changing significantly in #2889, so I'd hold off on migrating these buttons.

Comment (rhamilto) [reply frontend/public/components/sidebars/resource-sidebar.jsx]: Now included in #2889, so please drop the changes in this PR.

File: frontend/public/style/_layout.scss [hunk 2]
```diff
@@ -55,8 +55,8 @@ body,
 
   &__sidebar--hidden {
     position: absolute;
-    right: 23px;
-    top: 1px;
+    right: 13px;
+    top: -1px;
```
Comment (rhamilto) [inline frontend/public/style/_layout.scss]: Now included in #2889, so please drop the changes in this PR.

File: frontend/public/style/_overrides.scss [hunk 3]
```diff
@@ -135,6 +135,18 @@ h6 {
...
+  font-size: 21px !important;
+  line-height: 1;
+  opacity: .35;
+  padding: 0;
+  &:hover,
+  &:focus {
+    opacity: 1;
+  }
+}
+
```
Comment (rhamilto) [inline frontend/public/style/_overrides.scss]: Now included in #2889, so please drop the changes in this PR.

File: frontend/public/style/_layout.scss (hunk 2 above)
Comment (sg00dwin) [reply frontend/public/style/_layout.scss]: dropped

File: frontend/public/style/_overrides.scss (hunk 3 above)
Comment (sg00dwin) [reply frontend/public/style/_overrides.scss]: dropped

**Pull request merged**